### PR TITLE
Move safello config to @relayer

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-relayer/lib/config/relayer.ts [hunk 1]
```diff
@@ -14,6 +15,7 @@ export interface Config {
   contractWhiteList: ContractWhiteList;
   factoryAddress: string;
   supportedTokens: SupportedToken[];
+  safelloUrl: string;
```
Comment (marekkirejczyk) [inline universal-login-relayer/lib/config/relayer.ts]: `RelayerConfig` should not have `safelloUrl`, rather `safelloAppId`.

File: universal-login-ops/src/dev/startDevelopment.ts [hunk 2]
```diff
@@ -53,7 +53,12 @@ function getRelayerConfig(jsonRpcUrl: string, wallet: Wallet, walletMasterAddres
     contractWhiteList,
     factoryAddress,
     supportedTokens,
-    tokenContractAddress: tokenAddress
+    tokenContractAddress: tokenAddress,
+    safelloUrl: 'https://app.s4f3.io/sdk/quickbuy.html?appId=1234-5678&border=true' +
+    '&address-helper=true' +
+    '&lang=en' +
+    '&country=other' +
+    '&crypto=eth'
```
Comment (marekkirejczyk) [inline universal-login-ops/src/dev/startDevelopment.ts]: should be only `safelloAppId`

File: universal-login-relayer/lib/config/relayer.ts (hunk 1 above)
Comment (marekkirejczyk) [reply universal-login-relayer/lib/config/relayer.ts]: While we at it propose to add a separate section, like this:
```ts
export interface Config {
  onRamps: OnRampsConfiguration;
}

class OnRampsConfiguration {
  safelloAppId: '',
  ...
}
```

File: universal-login-commons/lib/core/models/relayer.ts [hunk 3]
```diff
@@ -19,4 +19,5 @@ export interface PublicRelayerConfig {
   factoryAddress: string;
   chainSpec: ChainSpec;
   contractWhiteList: ContractWhiteList;
+  safelloUrl: string;
```
Comment (marekkirejczyk) [inline universal-login-commons/lib/core/models/relayer.ts]: I would add separate section 'onRamps'.

File: universal-login-relayer/lib/config/safello.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  country: string;
+  crypto: string;
+}
+
+export const getSafelloUrl = (urlConfig: SafelloConfig) => 'https://app.s4f3.io/sdk/quickbuy.html?appId=1234-5678' +
+  '&border=true' +
+  '&address-helper=true' +
+  `&lang=${urlConfig.language}` +
+  `&country=${urlConfig.country}` +
+  `&crypto=${urlConfig.crypto}`;
```
Comment (marekkirejczyk) [inline universal-login-relayer/lib/config/safello.ts]: It should be sth more like this:
```ts
export const getSafelloUrl = (safelloConfig: SafelloConfig, request: express.Request, crypto = 'eth') => {
  const {appId, addressHelper} = safelloConfig;
  const {lang} = getLang(request);
  const country = getCountryFromIp(request.ip);
  return 
    `${baseUrl}?
       appId=${appId}&
       address-helper=${addressHelper}&
       lang=${language}&       
       country=${country}&
       crypto=${crypto}`
}
```

File: universal-login-sdk/lib/services/FutureWalletFactory.ts [hunk 5]
```diff
@@ -23,7 +23,7 @@ export type FutureWallet = {
 export class FutureWalletFactory {
 
   constructor(
-    private config: Omit<PublicRelayerConfig, 'chainSpec'>,
+    private config: Pick<PublicRelayerConfig, 'supportedTokens' | 'factoryAddress' | 'contractWhiteList' >,
```
Comment (marekkirejczyk) [inline universal-login-sdk/lib/services/FutureWalletFactory.ts]: ok, that is weird and at least deserves separate type

File: universal-login-relayer/lib/config/safello.ts [hunk 6]
```diff
@@ -0,0 +1,18 @@
+export const safelloUrlConfig = {
+  language: 'en',
+  country: 'other',
+  crypto: 'eth',
+};
+
+export interface SafelloConfig {
+  language: string;
+  country: string;
+  crypto: string;
```
Comment (marekkirejczyk) [inline universal-login-relayer/lib/config/safello.ts]: All 3 should be dynamic, see the comments below.

File: universal-login-commons/lib/core/models/relayer.ts (hunk 3 above)
Comment (xitronix) [reply universal-login-commons/lib/core/models/relayer.ts]: Config instead url now

File: universal-login-react/src/integration/Safello.tsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
+
+import React from 'react';
+import {LocalizationConfig, SafelloConfig} from '@universal-login/commons';
+
+interface Safello {
```
Comment (marekkirejczyk) [inline universal-login-react/src/integration/Safello.tsx]: SafelloProps

**Pull request merged**